source/deployment/sso-office.rst

```diff
@@ -0,0 +1,30 @@
+Office 365 Single Sign-On (E20)
+===============================
+
+Available in `Enterprise Edition E20 <https://mattermost.com/pricing-self-managed/>`__.
+
+.. include:: ../cloud/cloud-administration/common-sso-office365.rst
+
+Configure Mattermost ``config.json`` for Office 365 SSO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Instead of using the System Console, you can add the Office 365 settings directly to the ``config.json`` file directly on your Mattermost server.
```

Comment:
```suggestion
Instead of using the System Console, you can add the Office 365 settings directly to the ``config.json`` file on your Mattermost server.
```